packages/sql-mapper/test/helper.js

```diff
@@ -145,3 +145,46 @@ module.exports.setupDatabase = async function ({ seed, cache, t }) {
     cache
   })
 }
+
+function match (actual, expected) {
+  for (const key in expected) {
+    if (key in actual) {
+      if (typeof expected[key] === 'object' && expected[key] !== null) {
+        if (!match(actual[key], expected[key])) {
+          return false
+        }
+      } else {
+        if (actual[key] !== expected[key]) {
+          return false
+        }
+      }
+    } else {
+      return false
+    }
+  }
+  return true
+}
+
+module.exports.match = match
```

Comment:
This is better to be put inside utils module, I suspect this would be used often.